unit_tests/node_kernels/UnitTestScalarMassBDFNodeKernel.C

```diff
@@ -0,0 +1,72 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "kernels/UnitTestKernelUtils.h"
+#include "UnitTestUtils.h"
+#include "UnitTestHelperObjects.h"
+
+#include "node_kernels/ScalarMassBDFNodeKernel.h"
+
+#ifndef KOKKOS_ENABLE_CUDA
+namespace {
+namespace bdf2_golds {
+namespace scalar_mass {
+
+static constexpr double lhs[8][8] = {
+{0.10943331816113, 0, 0, 0, 0, 0, 0, 0, },
```

Comment:
I am pretty certain these gold values are incorrect, the numbers along the diagonal should be 

```
gamma1 * rhoNp1 * dualVolume * dt = 1.0 * 1.0 * 0.125 / 0.1 = 1.25
```

I think we have a bug somewhere that we need to track down.

Reply:
Mixture fraction deals with `density` field differently than other equation systems. So the previous comment is incorrect.